mappings/net/minecraft/client/render/block/entity/BlockEntityRenderer.mapping

```diff
@@ -5,9 +5,9 @@ CLASS net/minecraft/class_827 net/minecraft/client/render/block/entity/BlockEnti
 	METHOD method_3563 rendersOutsideBoundingBox (Lnet/minecraft/class_2586;)Z
 		ARG 1 blockEntity
 	METHOD method_3569 render (Lnet/minecraft/class_2586;FLnet/minecraft/class_4587;Lnet/minecraft/class_4597;II)V
-		ARG 1 blockEntity
-		ARG 2 tickDelta
+		ARG 1 entity
```

Comment:
```suggestion
		ARG 1 blockEntity
```
I prefer to keep the old block entity name here.

Reply:
If we can, I'd prefer that we keep it to one word parameter names wherever possible. Also this can just be called entity for consistency with all the other renderers. If you're unsure, there is the type declaration: `BlockEntity entity`.

Reply:
Sounds great. Keep the `entity` name.